package.json

```diff
@@ -18,19 +18,25 @@
       "prettier --write \"./**/*.{js,jsx,json,ts,tsx}\""
     ]
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
   "dependencies": {
     "@auth0/nextjs-auth0": "^1.3.1",
+    "@storybook/react": "^6.2.9",
     "@testing-library/jest-dom": "^5.12.0",
     "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^13.1.8",
     "@types/jest": "^26.0.23",
+    "@types/react": "^17.0.5",
     "babel-jest": "^26.6.3",
+    "husky": "^4.3.8",
```

Comment:
can me move husky to dev dependency since it's a dev tool?

Reply:
added to dev dependency